README.md

```
@@ -21,55 +21,82 @@ The QC metrics calculated by Picard tools and output by this app are informative
 
 The outputs of this app are to be displayed visually using [MultiQC](http://multiqc.info/), and assessed for inconsistencies across the alignment or variant-calling summaries. These summaries collectively contain the per-cycle base distribution, target enrichment, read duplication, and variant-calling statistics.
 
-## What data are required for this app to run?
-All inputs are optional from the perspective of DNANexus, which allows the app the flexibility of running QC on either alignments or variant calls. However, for successful exection, we list which data files are actually required for each mode to run. Bear in mind that some arguments are shared by alignment and variant calling (such as **fasta_index**), and as such care should be taken when running a complete assessment of the data.
+## What inputs are required for this app to run?
+All inputs are optional from the perspective of DNANexus, which allows the app the flexibility of running QC on either DNA alignment, RNA-seq, or variant calls (or combinations thereof). However, for successful execution, we list which data files are actually required for each mode to run. Bear in mind that some arguments are shared across modes (such as **fasta_index**), and as such care should be taken when running a complete assessment of the data.
+
+### Common arguments
+
+- **stringency (default=silent)**
+    - How the app responds when picard's internal BAM file validation returns warnings. Can be one of three settings:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)